src/Appwrite/Platform/Workers/Migrations.php

```
@@ -789,6 +789,7 @@ protected function sendCSVEmail(
             'terms' => $platform['termsUrl'],
             'privacy' => $platform['privacyUrl'],
             'platform' => $platform['platformName'],
+            'type' => $migration->getAttribute('destination', 'CSV'),
```

PHPMD (2.15.0)

[warning] Avoid unused local variables such as '$migration'. (undefined)

(UnusedLocalVariable)